tests/medium/test_ide.py

```diff
@@ -60,6 +60,70 @@ def test_install_with_changed_checksum_page(self):
         self.assertFalse(self.launcher_exists_and_is_pinned(self.desktop_filename))
 
 
+class EclipsePHPIDEInContainer(ContainerTests, test_ide.EclipsePHPIDETests):
```

Comment:
I guess the same can be applied here with inheritance + only changing the desired parameters ;)